android.pri

```diff
@@ -1,9 +1,26 @@
 include($$PWD/libs/qtandroidserialport/src/qtandroidserialport.pri)
 message("Adding Serial Java Classes")
 QT += androidextras
+
 ANDROID_PACKAGE_SOURCE_DIR = $$PWD/android
+exists($$PWD/custom/android) {
+    message("Merging $$PWD/custom/android/ -> $$PWD/android/")
+    ANDROID_PACKAGE_SOURCE_DIR = $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR
+
+    system($$QMAKE_MKDIR $$ANDROID_PACKAGE_SOURCE_DIR)
+    system($$QMAKE_COPY_DIR $$PWD/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR)
+    system($$QMAKE_COPY_DIR $$PWD/custom/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR)
+}
+
+exists($$PWD/custom/android/AndroidManifest.xml) {
+    OTHER_FILES += \
+    $$PWD/custom/android/AndroidManifest.xml
```

Comment:
This allows editing whichever AndroidManifest.xml in Qt Creator 
![custom_manifest](https://user-images.githubusercontent.com/8327961/89642778-bc1dbd80-d8b4-11ea-95e8-d9cd11956ff7.png)